mungegithub/mungers/approval-handler.go

```diff
@@ -231,7 +275,7 @@ func createApproverSet(comments []*githubapi.IssueComment) sets.String {
 	approverMatcher := c.CommandName(approveCommand)
 	for _, comment := range c.FilterComments(comments, approverMatcher) {
 		cmd := c.ParseCommand(comment)
-		if cmd.Arguments == "cancel" {
+		if cmd.Arguments == cancel {
```

Comment:
One question: what does `Delete` do if there is no such user in the set? i.e. what happens when someone cancels without approving in the first place?